### ISSUE TITLE
About rendering_options

### ISSUE THREAD

**Issue by ErisRolo** [hse1032/GSGAN#4]
Hi,thanks for your great work.
I found that in the appendix of the paper, the initial value of the scale is e^(-1/√N), but in the code, the 'scale_init' in 'custom_options' is set to log(1/√N). Even using np.exp(), it only results in 1/√N. Why is this the case? Looking forward to your answer.
​

**Comment by hse1032** [comment]
Hello, thanks for your interest in our paper!

We aim to initialize the Gaussian scale to have 1 / sqrt(N).
As the rasterization code in 3DGS contains exponential operation itself, we additionally apply log on the initialized scales.
(log(1/√N) of 'scale_init' in 'custom_options')
I think I made a mistake in the appendix, and please note that the code is the correct implementation.

By the way, any change in 'scale_init' should lead to changes in Gaussian scales.
You might check it by observing the rendered images at the start of training, which are saved in the log directory.

**Comment by ErisRolo** [comment]
Thank you so much for your reply!

I have another question: Since both EG3D and 3DGS camera coordinate systems are based on the OpenCV standard, why is the camera direction conversion matrix from EG3D to 3DGS defined as:
```        
self.reverse_camera_direction = torch.tensor(
                        [[1, 0, 0, 0],
                        [0, -1, 0, 0],
                        [0, 0, -1, 0],
                        [0, 0, 0, 1]],
                        dtype=torch.float32,
)
```

**Comment by ErisRolo** [comment]
If we consider the difference in world coordinate systems, EG3D's world coordinate system is a left-handed system where the x-axis points to the right and the z-axis points upwards, while 3DGS still uses OpenCV's standard right-handed system. I think the conversion matrix should be:
```
self.reverse_camera_direction = torch.tensor(
                        [[1, 0, 0, 0],
                        [0, 0, -1, 0],
                        [0, -1, 0, 0],
                        [0, 0, 0, 1]],
                        dtype=torch.float32,
```
I am not sure if my understanding is correct and would really appreciate your help on this matter.

**Comment by hse1032** [comment]
Sorry for the late reply.

There might be some bugs when converting the camera system between EG3D and 3DGS.

In this repository, I simply validated the changed camera parameters heading the origin of the world coordinate.
(In that case, I assume there would be no problem with training 3D GANs, as flip in the camera parameters can be modeled by flipping the Gaussians into the corresponding axis.)
